### ISSUE TITLE
Use more accurate pseudo GT for training but get reduced results 

### ISSUE THREAD

**Issue by chenhaolin1989** [clovaai/CRAFT-pytorch#78]
Hi, I use the model (craft_ic15_20k.pth) you released for IC15 pseudo GT prediction and then use such pseudo GT for IC15 training and test and I found the (hmean) performance dropped from 83.XX to 79.XX. However, if I use synthetic data trained model, I can get heman around 82.XX. I am confused that why I used more accurate pseudo GT for training but got worse performance? Thank you,

**Comment by YoungminBaek** [comment]
This is a quite interesting experiment.

I think the problem is the lack of the  using confidence map in our training scheme. Pseudo GT has many noises, so we need to use it more carefully. The confidence map is computed by comparing the number of the estimated characters and the length of GT transcription, and used for weighting the loss.

I guess that this may degenerate the performance with pseudo GT.
